css/form-display.css

```diff
@@ -0,0 +1,47 @@
+@media print {
+    .noprint {
+       visibility: hidden;
+    }
+    @page {
+        size: auto;   /* auto is the initial value */
+        margin: 0;  /* this affects the margin in the printer settings */
+    }
+ }
+
+footer {
+    background-color: #ffffff; 
+    opacity: 100%;
+    width: 189mm; /* Match .styBodyClass width to make this appear on the bottom of the form*/
+    display: flex; 
+    position: fixed; 
+    bottom: 0; 
+    left: 0;
+    align-items: center;
+    margin-left: 0;
```

Comment:
is `margin-left: 0` overriding from other CSS, or is it needed here if `left: 0` is above?

Reply:
`margin-left:0` is overriding `margin-left: 6px` is from the styBodyClass created by the render function ([code](https://github.com/newjersey/taxation-mef-viewer/blob/main/js/transform.js#L253)).

However, since we've made position `fixed`, I don't think the `margin-left` override is necessary. I tried removing the styling in browser and it didn't affect the position of the button. I'll remove it.